src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -57,9 +57,9 @@ contains the following details:
 | `--kubecontext` `<string>`            | kubeconfig context to use
 | `--repository` `<string>`             | The repository from where the various Submariner images will be sourced (default "quay.io/submariner")
 | `--version` `<string>`                | Image version
-| `--service-discovery`                 | Enable Multi Cluster Service Discovery (default true)
-| `--globalnet`                         | Enable support for overlapping Cluster/Service CIDRs in connecting clusters (default disabled)
+| `--components <strings>`              | Comma-separated list of components to be installed - any of `service-discovery`,`connectivity`,`globalnet`. The default is: `service-discovery`,`connectivity`
```

Comment:
I noticed in the code it says "_The default is all components_" which isn't correct and doesn't match here. While you're in the code changing the `globalnet` flag, can you update this as well?

Reply:
sure

Reply:
As pointed out on https://github.com/submariner-io/submariner-operator/pull/1228, specifying just `--components globalnet` disables `service-discovery` & `connectivity` so doesn't make much sense.

Reply:
Treating `globalnet` as a component seems problematic as it isn't standalone. Since we have the flag, I think we should remove it as a component. @sridhargaddam @mangelajo WDYT?

Reply:
I tend to agree with @tpantelis. It's like a sub-flag of connectivity. Just thinking out loud, what are your thoughts on "connectivity, connectivity-with-globalnet"?

Reply:
yeah we could do somethings like that. I think for v0.9 we should leave it as a flag, since it's pre-existing, but remove the `globalnet` component since that was introduced in v0.9.